django/cantusdb_project/main_app/views/chant.py

```diff
@@ -129,28 +129,32 @@ def get_feast_selector_options(source: Source) -> list[tuple[str, int, str]]:
     return deduped_folios_feasts_lists
 
 
-def get_chants_with_feasts(chants_in_folio: QuerySet) -> list:
-    # this will be a nested list of the following format:
-    # [
-    #   [feast_id_1, [chant, chant, ...]],
-    #   [feast_id_2, [chant, chant, ...]],
-    #   ...
-    # ]
+def get_chants_with_feasts(
+    chants_in_folio: QuerySet[Chant],
+) -> list[tuple[Optional[Feast], list[Chant]]]:
+    """
+    Takes a queryset of chants and returns a nested list
+    nested list of the following format:
+    [
+      [feast_id_1, [chant, chant, ...]],
+      [feast_id_2, [chant, chant, ...]],
+      ...
+    ]. The queryset of chants should have the related feast object prefetched.
+    """
```

Comment:
Since we're now returning a list of tuples, this docstring should reflect that (right now it shows a nested list). And can the following sentence at the end be moved to a new line?